CMakeLists.txt

```diff
@@ -239,7 +239,10 @@ endfunction()
 
 if(WITH_JAEGER)
   find_package(Thrift QUIET)
-  if(NOT Thrift_FOUND)
+  if(Thrift_FOUND)
+    find_package(Boost REQUIRED)
+    include_directories(${Boost_INCLUDE_DIRS})
```

Comment:
@ideepika  - Sorry for replying late on this. Wondering why do we need to explicitly add the Boost headers? Jaeger build is successful in our CI pipeline even without this.

Reply:
@ideepika  - Do you have any updates here ? Thanks.

Reply:
@lalitb sorry I must have missed the notification, I will check and post update tomorrow, thanks!